src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift

```diff
@@ -0,0 +1,133 @@
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+struct DeltaOnboardingV15ViewModel {
+	
+	// MARK: - Init
+
+	init(
+		supportedCountries: [Country]
+	) {
+		self.supportedCountries = supportedCountries.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
+	}
+
+	// MARK: - Internal
+
+	var dynamicTableViewModel: DynamicTableViewModel {
+		DynamicTableViewModel.with {
+			$0.add(
+				.section(
+					header: .image(
+						UIImage(named: "Illu_EUTracingOnboarding"),
+						accessibilityLabel: AppStrings.DeltaOnboarding.accImageLabel,
+						accessibilityIdentifier: AccessibilityIdentifiers.DeltaOnboarding.accImageDescription,
+						height: 250
+					),
+					cells: [
+						.title2(
+							text: AppStrings.DeltaOnboarding.title,
+							accessibilityIdentifier: AccessibilityIdentifiers.DeltaOnboarding.sectionTitle
+						),
+						.body(
+							text: AppStrings.DeltaOnboarding.description,
+							accessibilityIdentifier: AccessibilityIdentifiers.DeltaOnboarding.description
+						),
+						.body(
+							text: AppStrings.DeltaOnboarding.downloadInfo,
+							accessibilityIdentifier: AccessibilityIdentifiers.DeltaOnboarding.downloadInfo
+						),
+						.space(height: 16)
+					]
+				)
+			)
+			$0.add(
+				.section(
+					separators: .none,
+					cells: buildCountryLabel()
+				)
+			)
+			$0.add(
+				.section(
+					separators: .inBetween,
+					cells: supportedCountries.map {
+						DynamicCell.icon($0.flag, text: $0.localizedName, iconWidth: 28) { _, cell, _ in
+							cell.contentView.layoutMargins.left = 32
+							cell.contentView.layoutMargins.right = 32
+							cell.selectionStyle = .none
+						}
+					}
+				)
+			)
+			$0.add(
+				.section(
+					cells: [
+						.space(height: 20),
+						
+						.custom(
+							withIdentifier: DeltaOnboardingV15ViewController.CustomCellReuseIdentifiers.roundedCell,
+							configure: { _, cell, _ in
+								guard let cell = cell as? DynamicTableViewRoundedCell else { return }
+
+								cell.configure(
+									title: NSMutableAttributedString(
+										string: AppStrings.DeltaOnboarding.legalDataProcessingInfoTitle
+									),
+									body: NSMutableAttributedString(
+										string: AppStrings.DeltaOnboarding.legalDataProcessingInfoContent
+									),
+									textColor: .textPrimary1,
+									bgColor: .separator
+								)
+							}
+						)
+					]
+				)
+			)
+		}
+	}
+
+	// MARK: - Private
+
+	private let supportedCountries: [Country]
+
+	private func buildCountryLabel() -> [DynamicCell] {
```

Comment:
```suggestion
	private func buildCountryCell() -> [DynamicCell] {
```

Reply:
Also why does this return an array if we only return one cell?

Reply:
If supportedCountries.isEmpty there are 2 cells.